session-1-Local-Variable-Type-Inference/01_Primitive_Data_Types/README.md

```diff
@@ -0,0 +1,25 @@
+**Local Variable Type Inference for Primitive Data Types example**
+
+This first example is a class named `PrimitiveTypes` wichs its methods should use 
+`Type Inference` to declare variables using them in Expressions, Statements, and Blocks as Loops.
+
+Perform the below commands to see the contents of the respective Java classes contained in the src folder:
+
+    $ cat src/com/typeinference/PrimitiveTypes.java
+
+**Note:** in case one of the below .sh script fails due to the tree command, please take a look at Download and install the tree and wget command section in the README.md file and apply the appropriate solution.
```

Comment:
try to put keywords like `tree` etc in backticks (`) also `.sh` and README.md

Reply:
Done